TaylorFramework/Modules/finder/FinderExtensions.swift

```diff
@@ -14,14 +14,14 @@ extension FilePath {
     
     func deleteSuffixes() -> FilePath {
         var modifiedString = self
-        let suffixes = [DirectorySuffix.AllFiles, DirectorySuffix.Slash]
+        let suffixes = [DirectorySuffix.AllFiles.rawValue, DirectorySuffix.Slash.rawValue]
         for suffix in suffixes {
             modifiedString = modifiedString.deleteSuffix(suffix)
         }
         
-        if modifiedString.hasSuffix(DirectorySuffix.AnyCombination) &&
-            modifiedString.hasPrefix(DirectorySuffix.AnyCombination) {
-                modifiedString = modifiedString.stringByReplacingOccurrencesOfString(DirectorySuffix.AnyCombination, withString: String.Empty)
+        if modifiedString.hasSuffix(DirectorySuffix.AnyCombination.rawValue) &&
+            modifiedString.hasPrefix(DirectorySuffix.AnyCombination.rawValue) {
```

Comment:
Alias `DirectorySuffix.AnyCombination.rawValue` as you are overusing it in this function. Or declare another variable for it.